### DISCUSSION TITLE
Cannot play live streaming video on iOS

### DISCUSSION THREAD

**Discussion by 1psled** [mozilla/hubs#4023] [General Questions & Discussion] [answered]
Hello, I needed a live streaming video while using Mozilla Hub. I tried copying the live stream link directly into the video object and dragging the stream link within the Hub. VOD works well, but strangely, only stream links didn't work on iOS.
I'd like to ask if the stream link is playing on iOS.

**Comment by andybak** [comment]
Same here. This seems quite a major issue to remain unanswered for 7 days.

Is this across the board or are we doing something wrong?

Videos added in Spoke. Tried multiple streaming sources (YouTube, Vimeo JWPlayer).

Nothing works across multiple iOS devices.

**Comment by rufianeslabel** [comment]
Same here :(

**Comment by el-tech** [comment]
I tried hosting videos on my own serve via AWS S3, video is coming through but no sound.

**Comment by johnshaughnessy** [comment, marked as answer]
We've had a lot of issues with videos in safari and on iOS. I wish I could say we're close to fixing them but there seems to be a wide variety of problems that have been difficult to correctly diagnose and solve. 

There will likely not be any followup on this issue in this discussion, as there's already been debugging steps documented in existing issues. I'll point you to those so that you can add any new information you have that will help the dev team or volunteer contributors fix these problems.

https://github.com/mozilla/hubs/issues/2345
"Safari + video" issues: https://github.com/mozilla/hubs/issues?q=is%3Aissue+is%3Aopen+safari+video+
"iOS + video" issues: https://github.com/mozilla/hubs/issues?q=is%3Aissue+is%3Aopen+iOS+video+

Thank you for patience and I'm sorry I don't have a better solve at this time.

**Reply by 1psled** [reply]
Thx for the info !! 😁
